pkg/trace/info/info.go

```diff
@@ -259,11 +214,8 @@ func InitInfo(conf *config.AgentConfig) error {
 		expvar.Publish("uptime", expvar.Func(publishUptime))
 		expvar.Publish("version", expvar.Func(publishVersion))
 		expvar.Publish("receiver", expvar.Func(publishReceiverStats))
-		expvar.Publish("sampler", expvar.Func(publishSamplerInfo))
 		expvar.Publish("trace_writer", expvar.Func(publishTraceWriterInfo))
 		expvar.Publish("stats_writer", expvar.Func(publishStatsWriterInfo))
-		expvar.Publish("prioritysampler", expvar.Func(publishPrioritySamplerInfo))
```

Comment:
Why are we not publishing these anymore?

Reply:
Internal states of the samplers are obscure and don't bring valuable information.
If ever needed, with:
- the kept/seen counts added in the PRs
- customer config
- rates_by_service from expvars
^ We could simulate the internal states

Should I keep kept/seen counts in the expvars or metrics.Count are enough?

Reply:
Metrics should be fine. They're much better for debugging. I don't think many people check the expvars. They are however included in the flare.